aoc/main.py

```diff
@@ -0,0 +1,61 @@
+import praw
+import requests
+import os
+
+client_id = os.environ["REDDIT_CLIENT_ID"]
+client_secret = os.environ["REDDIT_CLIENT_SECRET"]
+reddit_pass = os.environ["REDDIT_PASSWORD"]
+username = os.environ["REDDIT_USERNAME"]
+user_agent = 'AdventOfCode Leaderboard Updater (by https://github.com/ni5arga/)'
+aoc_session_cookie = os.environ["AOC_SESSION_COOKIE"]
```

Comment:
LGTM 👌🏽 

Can you add a small README detailing how to get this cookie value?
[Is this the correct way](https://github.com/wimglenn/advent-of-code-wim/issues/1#issue-193321235), or any other trick?

Reply:
Done, added.